desktop/nodeManager.js

```diff
@@ -3,13 +3,26 @@ import os from 'os';
 import { ipcConsts } from '../app/vars';
 
 const child = require('child_process').execFile;
+const find = require('find-process');
 
 const osTargetNames = {
   Darwin: 'mac',
   Linux: 'linux',
   Windows_NT: 'windows'
 };
 
+const getPidByName = ({ name }) => {
+  return new Promise((resolve, reject) => {
+    try {
+      find('name', name).then((list) => {
+        resolve(list?.length && list[0].pid ? list[0].pid : null);
```

Comment:
valid return is always array and thus it always has length prop

Reply:
True, but I want to know if the length is larger than 0 as well.

Reply:
Ok, but '?' is useless here

Reply:
true